src/draw_instanced.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<title>webgl 2 samples - draw instances</title>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+	<style>
+		body {
+			color: #cccccc;
+			font-family: Monospace;
+			font-size: 13px;
+			text-align: center;
+			background-color: #050505;
+			margin: 0px;
+			overflow: hidden;
+		}
+		
+		#info {
+			position: absolute;
+			top: 0px;
+			width: 100%;
+			padding: 5px;
+		}
+		
+		a {
+			color: #0080ff;
+		}
+	</style>
+</head>
+
+<body>
+
+	<!--
+		<canvas id="c" width=window.innerWidth height=window.innerWidth></canvas>
+	-->
+	<div id="info">webgl 2 samples - draw instances</div>
+
+	<!-- WebGL 2 shaders -->
+    <script id="vs" type="x-shader/x-vertex">#version 300 es
+		precision highp float;
+		precision highp int;
+		layout(location=0) in vec2 pos;
+		flat out int Instance;
+		void main()
+		{
+			Instance = gl_InstanceID;
+			gl_Position = vec4(pos + vec2(float(gl_InstanceID)-0.5,0), 0, 1);
+		}
+    </script>
+
+    <script id="fs" type="x-shader/x-fragment">#version 300 es
+		precision highp float;
+		precision highp int;
+		
+		flat in int Instance;
+		out vec4 Color;
+		
+		void main()
+		{
+			Color = vec4(1.0 - float(Instance), 0.5, 0.0 + float(Instance), 1.0);
+		}
+    </script>
+
+
+
+	<script>
+		
+		function createShader(str, type) {
```

Comment:
There's a lot of whitespace.  Throughout can we use four spaces (not tabs) for indenting?

In general, our JavaScript and GLSL can follow the Cesium conventions: https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Documentation/Contributors/CodingGuide/README.md (ignore the Cesium-specific ones).